draft-ietf-httpbis-http2bis.xml

```diff
@@ -895,7 +895,7 @@ HTTP Frame {
             <t>
               An endpoint can perform this minimal processing for all streams that are in the
               "closed" state.  Endpoints MAY use other signals to detect that a peer has received
-              the frames that caused stream to become "closed" and treat receipt of any frame other
+              the frames that caused the stream to become "closed" and treat receipt of any frame other
```

Comment:
I'm also ambivalent about quoting "closed" here, since we usually only quote when talking about 'caused the stream to enter the "closed" state'

Reply:
Why not replace ‘become “closed”’ with ‘caused the stream to enter the “closed” state’ directly? It’s more verbose but it has the advantage of being unambiguous about what ‘become “closed”’ means.

Reply:
```suggestion
              the frames that caused the stream to enter the "closed" state and treat receipt of any frame other
```